recipes/travis-encrypt/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: travis-encrypt
+  version: "0.6.0"
+
+source:
+  git_url: https://github.com/mandeep/Travis-Encrypt.git
```

Comment:
Please build from pypi or the github release. See:

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos